lib/bike_brigade_web/live/rider_live/form_component.html.heex

```diff
@@ -22,12 +22,12 @@
       Pronouns
       <% end %>
       <div class="mt-1 rounded-md shadow-sm">
-        <%= text_input f, :pronouns, required: true, autocomplete: "pronounds", phx_debounce: "blur", class: "block w-full px-3 py-2 placeholder-gray-400 transition duration-150 ease-in-out border border-gray-300 rounded-md appearance-none focus:outline-none focus:ring-blue focus:border-blue-300 sm:text-sm sm:leading-5" %>
```

Comment:
Good catch. I did a little bit of research on this one and based on [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#values) and the [spec](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute) pronouns aren't actually an autocomplete attribute anyway so not sure if it will do what's intended even after the change.